### ISSUE TITLE
Language status animation increases size of status icon container

### ISSUE THREAD

**Issue by daviddossett** [microsoft/vscode#147940]
It seems like the warning/error wiggle is increasing the size of the icon container in some strange way. Note the extra space on the left side:

https://user-images.githubusercontent.com/25163139/164767575-241a0a18-e715-42ac-b291-e0c1a4ff507d.mp4

**Comment by jrieken** [comment]
> Note the extra space on the left side:

Sorry, not seeing it

**Comment by daviddossett** [comment]
Grabbed a screenshot from the video. Looks like some sort of padding left issue? 

<img width="224" alt="CleanShot 2022-04-25 at 09 09 03@2x" src="https://user-images.githubusercontent.com/25163139/165129288-55d4692c-38ec-49b8-be4e-56a2e16916ad.png">

**Comment by jrieken** [comment]
Now i get it. Thanks. I believe that's the sizing/padding of the container of any status bar item. I'll see if I can find a better node to highlight

**Comment by jrieken** [comment]
@bpasero Do you have an idea how to tackle this? The unbalanced padding is because this item is a friend to the language name item. This usually isn't very obvious, tho visible on hover, but becomes more visible during the background flash. Is there another element that's full height that I can animate?

**Comment by bpasero** [comment]
I think the way padding is setup right now was a compromise to have nice equal width (to both sides) hover over the indicator and then over the friend entry as well:

![image](https://user-images.githubusercontent.com/900690/165513138-93326d8c-1263-4b17-8b8b-7e2320ea2283.png)

There is unfortunately not really a equally-padded element to use because of this compromise...

**Comment by bpasero** [comment]
So I think the issue is the background color here that applies because that is the box that wiggles? Can we apply it only on the Codicon?

![image](https://user-images.githubusercontent.com/900690/165513424-18ca1ccd-7e75-42a6-87d6-967a5b77fcb1.png)

**Comment by jrieken** [comment]
> So I think the issue is the background color here that applies because that is the box that wiggles? Can we apply it only on the Codicon?

I had tried that but it doesn't look good. We want to wiggle the icon and flash the (whole) background

**Comment by daviddossett** [comment]
FWIW I actually updated that padding recently so the container didn't look lopsided on hover/focus. The margin-left shouldn't cause any issues here since a background-color wouldn't be applied to that space. 

As for the selector, I need to double check, but I don't recall having this issue when I grabbed this for my hacky demo:

`document.querySelector('.monaco-workbench .part.statusbar > .items-container > .statusbar-item.compact-right > :first-child');
`

It wouldn't be scoped enough, though. I'll take a closer look at this today to see if I can find another way.
